.github/workflows/integration.yml

```diff
@@ -67,7 +67,7 @@ jobs:
       run: ./gradlew lintFrontend
 
     - name: Build with Gradle
-      run: time ./gradlew clean checkstyleMain checkstyleTest test systemTest coverage
+      run: time ./gradlew clean checkstyleMain checkstyleTest zipReport test systemTest coverage
```

Comment:
Would not including `zipReport` actually affect anything?

Reply:
I put this here because I thought `Build preview website` may need it. Initially, zipReport would have been done as compileJava was entirely dependent on it. But in this case, with the general dependency removed, none of the tasks here would have called zipReport.

Reply:
Yea, it seems like a separate gradle run is done for `Build preview website`. Perhaps @dcshzj could confirm that there is no dependency here between the steps.